tools/internal/mdocspec/ocspec.go

```diff
@@ -121,6 +122,9 @@ func parseYAML(source []byte) (*ppb.OCPaths, *rpb.OCRPCs, error) {
 			MethodName: methods,
 		}
 	}
+	if !hasMethod {
+		return nil, nil, fmt.Errorf("mdocspec: YAML does not have least one RPC method specified")
```

Comment:
nit, but in test format, we use want before got?

```suggestion
		return nil, nil, fmt.Errorf("mdocspec: Want at least one RPC method in the YAML, but got 0.")
```

Reply:
In Go tests we generally have got before want: https://google.github.io/styleguide/go/decisions.html#got-before-want. However since this is not a test I didn't consider this point. Is there some other value in changing the wording here?

Reply:
Updated wording works for me!